ec2/spark_ec2.py

```diff
@@ -349,6 +350,15 @@ def launch_cluster(conn, opts, cluster_name):
     if opts.identity_file is None:
         print >> stderr, "ERROR: Must provide an identity file (-i) for ssh connections."
         sys.exit(1)
+
+    if not os.path.exists(opts.identify_file):
+        print >> stderr, "ERROR: The identity file(%s) doesn't exist." % opts.identity_file
+        sys.exit(1)
+
+    if oct(os.stat(opts.identity_file).st_mode)[-3:] != "600":
+        print >> stderr, "ERROR: Must set the permissions for the private key file to 600."
```

Comment:
Let's use consistent terminology. Since we are calling it "identity file" elsewhere, call it "identity file" here as well, as opposed to "private key file".